android/src/main/kotlin/com/isvisoft/flutter_screen_recording/FlutterScreenRecordingPlugin.kt

```diff
@@ -116,63 +112,64 @@ class FlutterScreenRecordingPlugin(
         }
     }
 
-    fun calculeResolution(metrics: DisplayMetrics) {
+    fun calculeResolution(screenSize:Point) {
```

Comment:
is now calculated by real screen size, which prevents crashing on Android devices like XiaoMi M8